src/components/11-infographics/chart.njk

```diff
@@ -0,0 +1,35 @@
+<div class="usa-chart" data-chart-type="bar"> <!-- char-type: bar, line -->
+  <table class="usa-chart-data" data-min="0" data-max="100" data-increment="20" data-grid="true"> <!-- all values optional -->
+
+    <!--
+    label datasets with header row
+    unnessisary unless multiple values are assigned
+    -->
+    <tr>
+      <td></td>
+      <th class="usa-chart-dataset">set one</th>
```

Comment:
In various places: can you add the appropriate scope attributes to identify your headers? See: https://webaim.org/techniques/tables/data#headers